ansible/playbooks/aix.yml

```diff
@@ -233,20 +233,33 @@
           register: java7
           tags: java7
 
-        - name: Create /usr/java7_64 if it does not exist
-          file:
-            path: /usr/java7_64
-            state: directory
-          when: java7.stat.isdir is not defined
-
         - name: Transfer and Extract Java7
           unarchive:
             src: /Vendor_Files/aix/openjdk-7u-aix.tar
-            dest: /usr/java7_64
+            dest: /usr
             remote_src: no
           when: java7.stat.isdir is not defined
           tags: java7
 
+        - name: Find java7 directory
+          find:
+            file_type: directory
+            paths: /usr
+            patterns: 'jdk7u*'
```

Comment:
Directory name is `j2sdk_image` on the tarball we're extracting. Also given that that particular directory name doesn't have the version number, we should probably just rename instead of symlink for java 7 only